shared/profile/container.js

```diff
@@ -52,6 +56,7 @@ class ProfileContainer extends React.PureComponent<EitherProps<Props>> {
 
 const mapStateToProps = (state: TypedState, {routeProps, routeState, routePath}: OwnProps) => {
   const myUsername = state.config.username
+  const myAccountID = WalletConstants.getDefaultAccountID(state)
```

Comment:
Can you add a TODO to get rid of this piping after CORE-8785 is done? that allows us to skip explicitly setting the `from` if we intend to send from the default account.